db/migrate/20160223195539_migrate_network_partner_participants_to_facilitators.rb

```diff
@@ -0,0 +1,34 @@
+class MigrateNetworkPartnerParticipantsToFacilitators < ActiveRecord::Migration
```

Comment:
I mean, if I run this migration locally to smoke test things and then go to another branch, would I have lost anything, and/or is it necessary for me to roll back my state?